components/rating/index.css

```
@@ -11,27 +11,48 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Rating {
-  --spectrum-icon-star-height: var(--spectrum-global-dimension-size-225);
-  --spectrum-icon-star-width: var(--spectrum-global-dimension-size-225);
+  /* Icon */
+  --spectrum-rating-icon-height: var(--spectrum-workflow-icon-size-100);
+  --spectrum-rating-icon-width: var(--spectrum-workflow-icon-size-100);
+  /* Focus ring */
+  --spectrum-rating-border-radius: var(--spectrum-corner-radius-100);
+  --spectrum-rating-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
+  --spectrum-rating-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
+  --spectrum-rating-focus-indicator-color: var(--spectrum-focus-indicator-color);
+  /* Spacing (top/bottom edge to icon) */
+  --spectrum-rating-icon-spacing-vertical-top: var(--spectrum-component-top-to-workflow-icon-100);
+  /* Indicator width */
+  --spectrum-rating-indicator-width: var(--spectrum-rating-indicator-width);
+  /* Indicator height */
+  --spectrum-rating-indicator-height: var(--spectrum-border-width-200);
+  /* Indicator spacing */
+  --spectrum-rating-indicator-spacing: var(--spectrum-rating-indicator-to-icon);
+  /* Colors */
+  /* selected + emphasized */
+  --spectrum-rating-emphasized-icon-color-default: var(--spectrum-accent-content-color-default);
+  --spectrum-rating-emphasized-icon-color-hover: var(--spectrum-accent-content-color-hover);
+  --spectrum-rating-emphasized-icon-color-down: var(--spectrum-accent-content-color-down);
+  --spectrum-rating-emphasized-icon-color-key-focus: var(--spectrum-accent-content-color-key-focus);
+  /* Disabled */
+  --spectrum-rating-icon-color-disabled: var(--spectrum-rating-icon-color-disabled);
+  --spectrum-rating-icon-count: var(--spectrum-rating-icon-count);
 }
 
-/* A solid star */
-%spectrum-Rating-filledStar {
-  .spectrum-Rating-starActive {
-    display: block;
-  }
-  .spectrum-Rating-starInactive {
-    display: none;
-  }
+/* Spacing (between icons) */
+.spectrum--medium {
```

```suggestion
.spectrum--medium {
```

This updates spacing based on platform size correct? @pfulton I didn't see anything on the [design page](https://spectrum.adobe.com/page/rating/) specific to this sizing so I'm wondering how we determine whether to attach a style to platform size or t-shirst size?

---

@venkateshjo I've addressed this, and the token is now available in the `@spectrum-css/tokens@7.5.0` release. I made a note of this below, in your `package.json` file. Once you update the `package.json` and run `yarn install`, you can delete lines `41 - 48`, as the `--spectrum-rating-icon-spacing` will now be coming from our `@spectrum-css/tokens` package.

Apologies if this is confusing, but we do this for the Spectrum Web Components team and these "custom" tokens.